dep/SFMT/CMakeLists.txt

```diff
@@ -54,7 +54,7 @@ else ()
     if (HAVE_NEON)
       target_compile_options(sfmt PRIVATE -mfpu=neon -ftree-vectorize)
     endif()
-  elseif (TRINITY_SYSTEM_PROCESSOR STREQUAL "arm64")
+  elseif (TRINITY_SYSTEM_PROCESSOR MATCHES "arm64|aarch64")
```

Comment:
this is wrong, TRINITY_SYSTEM_PROCESSOR will never be set to "aarch64"

that is the point of our custom variable, normalizing the processor name